File: 05613.py
a = int(input())
while True:
    ops = input()
    if ops == '=':
        break

    b = int(input())
    if ops == '+':
        a += b
    elif ops == '-':
        a -= b
    elif ops == '*':
        a *= b
    else:
        a //= b

print(a)

File: 05613.cpp
#include <iostream>
using namespace std;

int main() {
	int a; cin >> a;
	int b; char ops;
	while (1) {
		cin >> ops;
		if (ops == '=') break;

		cin >> b;
		switch (ops) {
		case '+':
			a += b;
			break;
		case '-':
			a -= b;
			break;
		case '*':
			a *= b;
			break;
		case '/':
			a /= b;
			break;
		}
	}

	cout << a << endl;
}
